src/main/kotlin/mate/academy/service/FigureSupplier.kt

```diff
@@ -0,0 +1,32 @@
+package mate.academy.service
+
+import mate.academy.Circle
+import mate.academy.Figure
+import mate.academy.IsoscelesTrapezoid
+import mate.academy.model.Color
+import mate.academy.Rectangle
+import mate.academy.RightTriangle
+import mate.academy.Square
+import kotlin.random.Random
+
+class FigureSupplier {
+
+    fun getRandomFigure(): Figure {
+        val colorSupplier = ColorSupplier()
```

Comment:
The `ColorSupplier` class is used here but not imported. Make sure to import `ColorSupplier` from the appropriate package to avoid runtime errors.